packages/interface/src/index.ts

```diff
@@ -0,0 +1,61 @@
+type Mnemonic = string
+type Address = string
+enum Chain {
+  BITCOIN,
+  ETHEREUM,
+  NEAR,
+  TERRA,
+  SOLANA
+}
+
+enum Network {
+  MAINNET,
+  TESTNET
+}
+
+enum Token {
+  NATIVE,
+  ERC20,
+  ERC721
+}
+
+enum Hardware {
+  LEDGER
+}
+
+export default interface IWalletConstructor {
+  new (mnemomic: Mnemonic, network: Network): IWallet
+}
+
+export interface IWallet {
+  _mnemonic: Mnemonic
+  _network: Network
+  addAccount(chain: Chain, hardware?: Hardware): Account
```

Comment:
We need to support the ability to enable/disable an account (chain)

Reply:
```
const wallet = new Wallet(mnemo, NETWORK.MAINNET)
const state = getClientState()

state.accounts.forEach(account => {
wallet.addAccount(account.chain, account.hardware)
})

const enabledBitcoinAccounts = wallet.getAccounts(Chain.BITCOIN).filter(account => {
return !state.disabledAccounts.includes(account.id)
})